commercetools-models/src/test/java/io/sphere/sdk/products/commands/ProductUpdateCommandIntegrationTest.java

```diff
@@ -8,7 +8,10 @@
 import io.sphere.sdk.commands.UpdateAction;
```

Comment:
I am not sure now about what is the idea of this `possibleToHackUpdateForStagedAndCurrent` test now, what it does, is to set the staged parameter to false by using the `StagedWrapper`, but now the update actions include the staged parameter.